gitops/generate_build.go

```diff
@@ -89,25 +90,26 @@ func GeneratePACRepository(component appstudiov1alpha1.Component, config map[str
 
 	isAppUsed := IsPaCApplicationConfigured(gitProvider, config)
 
-	var gitProviderConfig *pacv1alpha1.GitProvider = nil
+	var gitProviderConfig *pacv1alpha1.GitProvider = &pacv1alpha1.GitProvider{}
```

Comment:
Are you sure that passing empty config instead of `nil` won't change PaC behavior?

Reply:
I don't think it would make any difference but to be on the safe side I updated the PR and kept the default `gitProviderConfig` to be `nil`.